style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As required for the project, kindly make sure your `style.css` file includes one class with styles for your header i.e you should use a class to style your header instead of the `h1` tag then add that class in the `<h1>` tag(in your index.html) file to look like `<h1 class="class_name">Hello Microverse </h1>`. I know you can do that :muscle: :muscle:.